cli/cli/gateway.py

```diff
@@ -150,6 +159,61 @@ def get_routes(self) -> List[Route]:
 
         return routes
 
+    def get_consumers(self) -> List[Consumer]:
+        resp = self.session.get(url=self.consumers_url)
+        consumer_data = resp.json().get("data")
+        consumer_data.sort(key=lambda x: x["username"])
+
+        return [Consumer.from_json(c) for c in consumer_data]
+
+    def print_consumers(self) -> None:
+        consumers: List[Consumer] = self.get_consumers()
+
+        click.secho(f"{'USERNAME':<20}", bold=True)
+        for c in consumers:
+            click.secho(f"{c.username:<20}")
+
+    def delete_consumer(self, consumer_name: str):
+        consumer_url = f"{self.consumers_url}/{consumer_name}"
+        resp = self.session.delete(url=consumer_url)
+        resp.raise_for_status()
+
+    def add_consumer(self, consumer_name):
+        consumer = Consumer(username=consumer_name)
+        resp = self.session.post(url=self.consumers_url, json=consumer.json())
+        resp.raise_for_status()
+
+    def add_jwt_cred(self, consumer_name: str) -> JwtCredential:
+        jwt_url = f"{self.consumers_url}/{consumer_name}/jwt"
+
+        resp = self.session.post(
+            url=jwt_url,
+            headers={"Content-Type": "application/x-www-form-urlencoded"},
+        )
+        resp.raise_for_status()
+
+        return JwtCredential.from_json(resp.json())
+
+    def get_jwt_creds(self, consumer_name: str) -> List[JwtCredential]:
+        jwt_url = f"{self.consumers_url}/{consumer_name}/jwt"
+
+        resp = self.session.get(url=jwt_url)
+        resp.raise_for_status()
+
+        creds_data = resp.json()["data"]
+        return [JwtCredential.from_json(d) for d in creds_data]
+
+    def print_jwt_cred(self, cred: JwtCredential, header=True):
+        if header:
+            click.secho(f"{'ALGORITHM':<15} {'SECRET':<40} {'ISS':<40}", bold=True)
+        click.secho(f"{cred.algorithm:<15} {cred.secret:<40} {cred.iss:<40}")
+
+    def print_jwt_creds(self, consumer_name):
+        creds = self.get_jwt_creds(consumer_name)
+
```

Comment:
I think we could also print the header in print_jwt_creds with an if in case there are no creds to avoid the needs of passing a header boolean. But either way is fine :1st_place_medal: 

```suggestion
if creds:
    click.secho(f"{'ALGORITHM':<15} {'SECRET':<40} {'ISS':<40}", bold=True)
```